packages/esm-patient-notes-app/src/notes/visit-notes-form.component.tsx

```diff
@@ -202,7 +265,7 @@ const VisitNotesForm: React.FC<DefaultWorkspaceProps> = ({ closeWorkspace, patie
       mutate,
       patientUuid,
       providerUuid,
-      selectedDiagnoses,
```

Comment:
It seems like we need `selectedPrimaryDiagnoses` as a dependency here, since it's referenced in part of this massive hook... (line 189)